webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -31,6 +37,7 @@ interface FormInputs {
   useAgentUser: boolean;
   useCheckpointStorage: boolean;
   workspaceName: string;
+  [cluster: string]: string | boolean | number | undefined;
```

Comment:
We know our bindings will only be string to string, so we can make our lives easier by putting the bindings under a separate field:
```suggestion
  bindings?: Record<string, string>;
```

Reply:
This is actually really helpful! Thank you